tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -244,4 +251,28 @@ describe(__filename, () => {
 
     expect(root.find('.SearchFilters-AddonType')).toHaveLength(0);
   });
+
+  it('sets themes filters shelf with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+    const root = render({ _config: fakeConfig });
+    const selectFilters = root.find(Select);
+
+    const optionValues = selectFilters
+      .find('#SearchFilters-AddonType')
```

Comment:
Oh, I only suggested this because it would allow you to do the find without having to initialize it with `Select`:

```diff
diff --git a/tests/unit/amo/components/TestSearchFilters.js b/tests/unit/amo/components/TestSearchFilters.js
index 635946348..c04db3627 100644
--- a/tests/unit/amo/components/TestSearchFilters.js
+++ b/tests/unit/amo/components/TestSearchFilters.js
@@ -255,10 +255,9 @@ describe(__filename, () => {
   it('sets themes filters shelf with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
     const fakeConfig = getFakeConfig({ enableStaticThemes: true });
     const root = render({ _config: fakeConfig });
-    const selectFilters = root.find(Select);
 
-    const optionValues = selectFilters
-      .find('.SearchFilters-AddonType')
+    const optionValues = root
+      .find('#SearchFilters-AddonType')
       .children()
       .map((option) => option.props().value);
     expect(optionValues).toContain(ADDON_TYPE_THEMES_FILTER);
```

Again, it's not a big deal and I know we are tagging soon 😀